test/PushComm/unit_tests/SubscribeBySig/SubscribeBySig.t.sol

```diff
@@ -0,0 +1,294 @@
+pragma solidity ^0.8.0;
+
+import { BaseTest } from "../../../BaseTest.t.sol";
+import { CoreTypes } from "../../../../contracts/libraries/DataTypes.sol";
+import { SignatureVerifier } from "contracts/mocks/MockERC1271.sol";
+
+import "forge-std/console.sol";
+
+contract SubscribeBySig_Test is BaseTest {
+    bytes constant _testChannelIdentity = bytes("test-channel-hello-world");
+
+    event Subscribe(address indexed channel, address indexed user);
```

Comment:
these events should've been added in test/utils/Events.sol file under CommEvents